helpers/calendar/ratios_and_utilization.py

```diff
@@ -95,29 +101,32 @@ def retrieve_cancellations_and_checkins_for_room(self,
                 checkins,cancellations, output
         """
         all_events = CommonAnalytics.get_all_events_in_a_room(
-                self, room_id, start_date, day_after_end_date)
+            self, room_id, start_date, day_after_end_date)
         if all_events:
             for event in all_events:
                 if event['participants']:
                     calendar_events_list.append(event)
         room_id = RoomModel.query.filter_by(id=room_id).first().id
 
-        checkins += (Events.query.filter(Events.room_id==room_id, Events.checked_in==True, Events.start_time>=start_date, Events.end_time<day_after_end_date)).count()  # noqa
+        checkins += (Events.query.filter(Events.room_id == room_id, Events.checked_in == True, Events.start_time >= start_date, Events.end_time < day_after_end_date)).count()  # noqa
 
-        cancellations += (Events.query.filter(Events.room_id==room_id, Events.cancelled==True, Events.start_time>=start_date, Events.end_time<day_after_end_date)).count()  # noqa
+        cancellations += (Events.query.filter(Events.room_id == room_id, Events.cancelled == True, Events.start_time >= start_date, Events.end_time < day_after_end_date)).count()  # noqa
 
         return checkins, cancellations, calendar_events_list, app_events_list
 
-    def map_results_to_ratio_class(self,
-                                   checkins,
-                                   cancellations,
-                                   calendar_events_list,
-                                   app_events_list,
-                                   room_name=None):
+    def map_results_to_ratio_class(self, **kwargs):
         """ Maps the checkins and cancellations to the ratio object
         :params
-            - checkins, cancellations, output, room_name
+            - checkins, cancellations, calendar_events_list, app_events_list,
+            room_name, room_id
         """
+        checkins = kwargs.get('checkins')
```

Comment:
I don't think these variable declarations are necessary here.

Reply:
I wanted to make the implementation clean by declaring the relevant variables. Not declaring the variables there would have me writing a lot of `kwargs.get('****')` in the code. If you feel this isn't an optimal way kindly suggest a better way of going about it.

Reply:
If you are not using these variables elsewhere other than in the `RatioOfCheckinsAndCancellations` method, then you dont have to declare them here, just use `kwargs.get()` in the method.

Reply:
A couple of them are being used in various instances hence the need for the declaration. I'll update the code to use `kwargs.get()` for those that are utilized in just one instance.